packages/venia-ui/lib/components/CheckoutPage/checkoutPage.css

```diff
@@ -102,6 +111,15 @@
         top: 5.5rem;
         height: min-content;
     }
+
+    .signin_container ~ .summaryContainer,
+    .formErrors ~ .summaryContainer {
+        grid-row: 3 / span 3;
+    }
+
+    .signin_container + .formErrors ~ .summaryContainer {
+        grid-row: 4 / span 3;
+    }
```

Comment:
This CSS feels a bit off to me - the summary container and the "guest checkout" text have a big gap. Was there a reason you added this? I know we did not provide desktop mocks.

[![Image from Gyazo](https://i.gyazo.com/f327eb135f36c465d7de83cabf5bc6dc.png)](https://gyazo.com/f327eb135f36c465d7de83cabf5bc6dc)

Reply:
I went ahead and removed it. Looks better:

[![Image from Gyazo](https://i.gyazo.com/c5fa007279485a3724c53f9d2ae06789.gif)](https://gyazo.com/c5fa007279485a3724c53f9d2ae06789)

Reply:
Actually tried to keep aligned with shipping info section as sign in will go away for logged in users. But the way you did looks fine.